src/web/mjs/connectors/BilibiliComics.mjs

```diff
@@ -37,7 +37,48 @@ export default class BilibiliComics extends BilibiliManhua {
                 ],
                 value: 'en'
             },
+            picquality: {
```

Comment:
This settings seems redundant, users always ask for the best possible image quality and resolution.

Reply:
1) Until we find someone who ask for lesser and this needs to be modified again (idk, for bandwidth/data limits reasons). 

2) If the website give the the choice you can be sure someone will ask ^^

3) Since the needed fix is impacting picture quality i thought it was a good idea to give a setting. Its just a setting you know. You can remove the setting but the math should stay (or be simplified to "if size in json <0 pic.quality = "good")